coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -3754,6 +3758,14 @@ def randint(context, node):
     rand_int = mb.cast(x=rand_uniform, dtype="int32", name=node.name)
     context.add(rand_int)
 
+@register_torch_op
+def rand(context, node):
+    shape, _, dtype, _, _ = _get_inputs(context, node, min_expected=1)
```

Comment:
I suggest removing the `min_expected=1`. That's not really doing anything here. The left hand side of the equals sign is assuming there is five outputs, i.e. you'll get an error if the `_get_inputs` call doesn't return five elements.